wiki/Tournaments/MWC/2018/7K/en.md

```diff
@@ -56,31 +56,19 @@ The osu!mania 7K World Cup 2018 is run by various community members by distribut
 
 ------------------------------------------------------------------------
 
-## Match Schedule: Semifinals
+## Match Schedule: Finals
 
-### Saturday, 3. March 2018
+### Sunday, 11. March 2018
```

Comment:
2018-03-11

Reply:
surprised that his is pointed out on the finals but not on previous rounds 🤔

Reply:
ye should on all of them, forgot about it 🤔